tests/tasks/aws/test_secrets_manager.py

```diff
@@ -23,3 +33,28 @@ def test_raises_if_secret_not_eventually_provided(self):
 
         with pytest.raises(ValueError, match="secret"):
             task.run()
+
+    def test_retrieve_plaintext_secret(self, mocked_boto_client):
+        def mocked_response(*args, **kwargs):
+            return {
+                "SecretString": '{"top-secret-string"}',
+            }
+
+        task = AWSSecretsManager(secret="test")
+        mocked_boto_client.get_secret_value.side_effect = mocked_response
+        returned_data = task.run()
+
+        assert returned_data == mocked_response()["SecretString"]
```

Comment:
My personal preference is returning "top-secret-string", but I wasn't sure if there were people out there doing wacky things with secrets and parsing it here could break that. Your question hints you'd might prefer "top-secret-string"? In which case I'd happily make the change now.

Reply:
Is there a reason you'd set the secret in AWS to `'{"top-secret-string"}'` instead of `"top-secret-string"` directly? Is that something AWS forces? I'm just not familiar with what they'll let you do and what payloads we should be expecting.

Reply:
This is just something that AWS forces and what boto3 will return when you call `get-secret-value`.

Reply:
My bad, perhaps just a peculiarity with our Terraform setup.